apps/preview/next/pages/showcases/SelectDropdown/SelectDropdownDisabled.tsx

```diff
@@ -0,0 +1,136 @@
+import { ShowcasePageLayout } from '../../showcases';
+// #region source
+import { useId, useRef, useState, type KeyboardEvent } from 'react';
+import classNames from 'classnames';
+import {
+  SfIconExpandMore,
+  SfListItem,
+  useDisclosure,
+  useDropdown,
+  SfIconCheck,
+  useTrapFocus,
+} from '@storefront-ui/react';
+
+type SelectOption = {
+  label: string;
+  value: string;
+};
+
+const options: SelectOption[] = [
+  {
+    label: 'Today',
+    value: 'today',
+  },
+  {
+    label: 'Tomorrow',
+    value: 'tomorrow',
+  },
+  {
+    label: 'Anytime',
+    value: 'anytime',
+  },
+];
+
+export default function SelectDropdownDisabled() {
+  const { close, toggle, isOpen } = useDisclosure({ initialValue: false });
+  const [selectedOption, setSelectedOption] = useState<SelectOption>();
+  const id = useId();
+  const listboxId = `select-dropdown-${id}`;
+  const selectTriggerRef = useRef<HTMLDivElement>(null);
+  const isDisabled = true;
+
+  const { refs, style: dropdownStyle } = useDropdown({ isOpen, onClose: close });
+
+  useTrapFocus(refs.floating, {
+    arrowKeysOn: true,
+    activeState: isOpen,
+    initialFocusContainerFallback: true,
+  });
+
+  const selectOption = (option: SelectOption) => {
+    setSelectedOption(option);
+    close();
+    selectTriggerRef.current?.focus();
+  };
+
+  const handleTriggerKeyDown = (e: KeyboardEvent<HTMLDivElement>) => {
+    if (e.key === ' ') toggle();
+  };
```

Comment:
```suggestion
  const handleTriggerKeyDown = (event: KeyboardEvent<HTMLDivElement>) => {
    if (event.key === ' ') toggle();
  };
```

shouldnt it bemaybe better use `e.code === "Space"` wdyt?

Reply:
[event.code unfortunately has small support (does not work on Chrome on Android)](https://caniuse.com/keyboardevent-code). I could use `event.which` or `event.charCode`, but these two are outside of any spec and typescript typings mark them as deprecated

Reply:
okay then leave `' '` if it has better support, thanks :)